pkgs/development/libraries/nss/generic.nix

```diff
@@ -103,6 +103,13 @@ stdenv.mkDerivation rec {
       # yes, this is correct. nixpkgs uses "host" for the platform the binary will run on whereas nss uses "host" for the platform that the build is running on
       target = getArch stdenv.hostPlatform;
       host = getArch stdenv.buildPlatform;
+      host_system =
```

Comment:
I think we should either
- rename the variables above, and move down the comment,
- rename `host_system` to `target_system`.

Also, it looks like we could use `stdenv.hostPlatform.uname.system` for this?